istioctl/cmd/istioctl/testdata/inject-config.yaml

```diff
@@ -0,0 +1,135 @@
+policy: enabled
```

Comment:
This is likely to get out of sync with the canonical injection config. We could probably get by with a simplified template which would make it easier to maintain and spot errors later on.

Reply:
Do you have a simplified injection config template?

I needed an injection config to get reproducible output and to test the `--injectConfigFile` CLI option.  I couldn't find a test example.

Reply:
https://github.com/istio/istio/blob/master/pilot/pkg/kube/inject/testdata/TestWebhookInject_template.yaml is a reasonable example to follow.